pkg/kubelet/server.go

```diff
@@ -367,7 +367,7 @@ func (s *Server) dockerHealthCheck(req *http.Request) error {
 		return errors.New("unknown Docker version")
 	}
 	// Verify the docker version.
-	result, err := version.Compare("1.15")
+	result, err := version.Compare("1.18")
```

Comment:
We should rename `version` to `apiVersion`, or maybe add some comments. 
I assume you ran docker 1.6 to get this api version @dchen1107.